src/components/DayPickerRangeController.jsx

```diff
@@ -170,6 +176,7 @@ export default class DayPickerRangeController extends React.Component {
       'last-in-range': day => this.isLastInRange(day),
       hovered: day => this.isHovered(day),
       'hovered-span': day => this.isInHoveredSpan(day),
+      'hovered-offset': day => this.isInHoveredSpan(day),
```

Comment:
Do we use the same method for hovered-span and for hovered-offset? does that cause any issues?